layout/theme.liquid

```diff
@@ -123,7 +123,7 @@
           </div>
         </div>
         {% else %}
-          {% unless current_user.is_anonymous? %}
+          {% if current_user.is_wholesale? %}

```

Comment:
@mauro-oto I understand. Let's change it to this:

`if current_user.is_wholesale? || current_user.is_customer? || current_user.is_admin?`